### PR TITLE
init project

### PR DESCRIPTION
**Pull request by Odinsees**
create project Next + TS, Eslint + Prettier
install start dependencies, create start architecture

### CODE REVIEW

File: src/components/common/footer/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import Link from 'next/link';
+
+import { FaceBookIcon } from './components/icons/facebook-icon/FacebookIcon';
+import { InstagramIcon } from './components/icons/instagram-icon/InstagramIcon';
+import { TwitterIcon } from './components/icons/twitter-icon/TwitterIcon';
+
+import style from './style.module.scss';
```
Comment (Ilya-Valasiuk) [inline src/components/common/footer/Footer.tsx]: `import styles from './styles.module.scss';`

File: src/components/common/footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+export const Footer = () => {
```
Comment (Ilya-Valasiuk) [inline src/components/common/footer/Footer.tsx]: `export const Footer: FunctionalComponent = () => {`

File: src/components/common/footer/components/icons/facebook-icon/FacebookIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { SVGProps, useState } from 'react'
+
+export const FaceBookIcon = (props: SVGProps<SVGSVGElement>) => {
+  const [active, setActive] = useState(false)
+  return (
+    <svg
+      onMouseEnter={() => setActive(true)}
+      onMouseLeave={() => setActive(false)}
```
Comment (Ilya-Valasiuk) [inline src/components/common/footer/components/icons/facebook-icon/FacebookIcon.tsx]: `hover` in css

File: src/components/common/footer/components/icons/instagram-icon/InstagramIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import * as React from 'react'
+import { SVGProps, useState } from 'react'
+
+export const InstagramIcon = (props: SVGProps<SVGSVGElement>) => {
+  const [active, setActive] = useState(false)
+  return (
+    <svg
+      onMouseEnter={() => setActive(true)}
+      onMouseLeave={() => setActive(false)}
```
Comment (Ilya-Valasiuk) [inline src/components/common/footer/components/icons/instagram-icon/InstagramIcon.tsx]: the same

File: src/components/common/footer/components/icons/twitter-icon/TwitterIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { SVGProps, useState } from 'react'
+
+export const TwitterIcon = () => {
+  const [active, setActive] = useState(false)
+  return (
+    <svg
+      onMouseEnter={() => setActive(true)}
+      onMouseLeave={() => setActive(false)}
```
Comment (Ilya-Valasiuk) [inline src/components/common/footer/components/icons/twitter-icon/TwitterIcon.tsx]: the same

File: src/components/common/footer/style.module.scss [hunk 6]
```diff
@@ -0,0 +1,88 @@
+@import '@styles/scss/abstracts/mixins';
+@import '@styles/scss/abstracts/variables';
+
+.container {
+  max-width: 1440px;
+  min-height: 64px;
+  margin-top: 20px;
+  .inner {
```
Comment (Ilya-Valasiuk) [inline src/components/common/footer/style.module.scss]:
```
.container {
  .class1 {}

  .class2 {}

  .class3 {}
}

```

File: src/components/common/header/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,102 @@
+import { useState } from 'react';
+
+import Link from 'next/link';
+
+import classNames from 'classnames';
+
+import style from './style.module.scss';
```
Comment (Ilya-Valasiuk) [inline src/components/common/header/Header.tsx]: is should be 
```
import { useState } from 'react';
import Link from 'next/link';
import classNames from 'classnames';

import styles from './styles.module.scss';
```

File: src/components/common/header/Header.tsx [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+export const Header = () => {
+  const [activeLink, setActiveLink] = useState<string | null>('home');
```
Comment (Ilya-Valasiuk) [inline src/components/common/header/Header.tsx:11]: To discuss

File: src/components/common/header/style.module.scss [hunk 9]
```diff
@@ -0,0 +1,122 @@
+@import '@styles/scss/abstracts/mixins';
+@import '@styles/scss/abstracts/variables';
+
+.headerContainer {
```
Comment (Ilya-Valasiuk) [inline src/components/common/header/style.module.scss]: the same here 
```
```
.container {
  .class1 {}

  .class2 {}

  .class3 {}
}

```
```

File: src/components/common/header/Header.tsx [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+  const [openBurger, setOpenBurger] = useState(false);
+
+  return (
+    <div className={style.headerContainer}>
```
Comment (Ilya-Valasiuk) [inline src/components/common/header/Header.tsx]: why `headerContainer`? 
why not just `header`

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 11]
```diff
@@ -0,0 +1,54 @@
+import { FunctionComponent } from 'react';
+
+import { Story } from '@models/story';
+
+import { useStories } from '@hooks/useStories';
+
+import Moment from 'moment';
+import Image from 'next/image';
+import styles from './styles.module.scss';
+import Link from 'next/link';
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: https://docs.google.com/document/d/1ez__g_pUi-T5aRnsBxumdu-stTuk6xZ6PtZroAd5BeY/edit

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+export const StoriesList: FunctionComponent = () => {
+  const formatDate = (date: Date) => {
+    const formatDate = Moment(date).format('MMM DD, YYYY, HH:mm');
+    return formatDate;
+  };
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: do we need to have this function in render?

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+
+  const { stories, isLoading, isError } = useStories('technology');
+  if (isLoading) return <div>Loading...</div>;
+  if (isError) return <div>Error</div>;
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]:
```
const { stories, isLoading, isError } = useStories('technology');

if (isLoading) return <div>Loading...</div>;

if (isError) return <div>Error</div>;
```

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+  return (
+    <div className={styles.container}>
+      {stories.map((story: Story, index: number) => {
+        console.log(story);
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: do we need it here?

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+    return formatDate;
+  };
+
+  const { stories, isLoading, isError } = useStories('technology');
+  if (isLoading) return <div>Loading...</div>;
+  if (isError) return <div>Error</div>;
+
+  return (
+    <div className={styles.container}>
+      {stories.map((story: Story, index: number) => {
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: do we need to assign `Story` and `number` type here?

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+        console.log(story);
+
+        return (
+          <Link key={index} href={`/story/`} passHref>
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: can we use another key?

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+            <a>
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: Do we need this `a` tag? What will be if we remove it?

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+              <div className={styles.storyContainer}>
+                <div className={styles.storyInfo}>
+                  <p className={styles.storySection}>{story.section}</p>
+                  <h2 className={styles.storyTitle}>{story.title}</h2>
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]: do we need here h2?

File: src/hooks/useStories.ts [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+
+type StoryFetcher = {
+  stories: Story[];
+  isLoading: boolean;
+  isError: boolean;
+};
+
+export const useStories = (section: string): StoryFetcher => {
+  const { data, error } = useSWR(
+    `${TOP_STORIES}/${section}.json?api-key=${API_KEY}`,
```
Comment (Ilya-Valasiuk) [inline src/hooks/useStories.ts]: api key should be added in fetcher method. We don't want to have api key in cache

File: src/pages/_app.tsx [hunk 20]
```diff
@@ -0,0 +1,24 @@
...
+import Head from 'next/head'
+import '@styles/scss/main.scss'
+
+function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <>
+      <Head>
+        <meta name="viewport" content="width=device-width" />
+      </Head>
+      <div className={'layout'}>
```
Comment (Ilya-Valasiuk) [inline src/pages/_app.tsx]:
```
<div className="layout">
```

File: src/service/storyFetcher.ts [hunk 21]
```diff
@@ -0,0 +1,5 @@
+import fetch from 'unfetch';
+
+export const storyFetcher = async (url: string) => {
+  return fetch(url).then(response => response.json());
```
Comment (Ilya-Valasiuk) [inline src/service/storyFetcher.ts]: here we should add api key to url

File: src/styles/scss/abstracts/_mixins.scss [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+  box-shadow: 0px 0px 10px rgba(22, 22, 22, 0.1);
+}
+
+@mixin extra-small-text-regular {
+  font-family: $type-family;
+  font-weight: $type-weight;
+  font-style: normal;
+  font-size: 14px;
+  line-height: 18px;
+  color: $text;
```
Comment (Ilya-Valasiuk) [inline src/styles/scss/abstracts/_mixins.scss]: usually  we want to have possibly to use another color in text. 
Let's add possibility to pass custom color to mixin

File: src/styles/scss/abstracts/_mixins.scss [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+@mixin heading-h2 {
+  font-family: $type-family;
+  font-weight: $type-weight--semi-bold;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: $text;
```
Comment (Ilya-Valasiuk) [inline src/styles/scss/abstracts/_mixins.scss]: Looks like all props are the same, just different values. Probably you can create general mixin here and use it 

Something like
```
@mixin custom-text (...params) {
  font-family:...
  font-weight: ...
  font-style: ...
...
}
```

File: src/styles/scss/base/_base.scss [hunk 24]
```diff
@@ -0,0 +1,66 @@
+* {
+  all: unset;
+  -webkit-text-fill-color: initial;
+  display: revert;
+}
```
Comment (Ilya-Valasiuk) [inline src/styles/scss/base/_base.scss]: what is that? where you took it?

File: src/styles/scss/layout/layout.scss [hunk 25]
```diff
@@ -0,0 +1,23 @@
...
+    .container {
+      max-width: 750px;
+    }
+  }
+  @media (max-width: 767px) {
+    .container {
+      max-width: none;
+    }
+  }
+}
```
Comment (Ilya-Valasiuk) [inline src/styles/scss/layout/layout.scss]: Please look at this code
```
.layout {
  max-width: 1440px;
  margin: 0 auto;
}

.container {
  max-width: none;
  margin: 0 auto;

  @media (min-width: 768px) {
    max-width: 750px;
  }

  @media (min-width: 992px) {
    max-width: 970px;
  }

  @media (min-width: 1200px) {
    max-width: 1200px;
  }
}
```

1. We don't have nesting
2. we don't have duplication
3. our styles should start from mobile and up. We using mobile first approach.

File: tsconfig.json [hunk 26]
```diff
@@ -0,0 +1,37 @@
...
+      "@service/*": ["src/service/*"],
+      "@utils/*": ["src/utils/*"],
+      "@utils": ["src/utils"],
+      "@models/*": ["src/models/*"],
+      "@models": ["src/models"],
+      "@styles/*": ["src/styles/*"],
+      "@styles": ["src/styles"],
+      "@hooks": ["src/hooks"],
+      "@hooks/*": ["src/hooks/*"]
+    }
```
Comment (Ilya-Valasiuk) [inline tsconfig.json]: why not just
```
"baseUrl": "./src",
```
?

**Review by Ilya-Valasiuk — CHANGES_REQUESTED** [review]

File: src/components/common/footer/Footer.tsx (hunk 1 above)
Comment (Odinsees) [reply src/components/common/footer/Footer.tsx]: fix naming and import

File: src/components/common/footer/Footer.tsx (hunk 2 above)
Comment (Odinsees) [reply src/components/common/footer/Footer.tsx]: fix this

File: src/components/common/footer/components/icons/facebook-icon/FacebookIcon.tsx (hunk 3 above)
Comment (Odinsees) [reply src/components/common/footer/components/icons/facebook-icon/FacebookIcon.tsx]: in progress

File: src/components/common/footer/components/icons/instagram-icon/InstagramIcon.tsx (hunk 4 above)
Comment (Odinsees) [reply src/components/common/footer/components/icons/instagram-icon/InstagramIcon.tsx]: in progress

File: src/components/common/footer/components/icons/twitter-icon/TwitterIcon.tsx (hunk 5 above)
Comment (Odinsees) [reply src/components/common/footer/components/icons/twitter-icon/TwitterIcon.tsx]: in progress

File: src/styles/scss/abstracts/_mixins.scss (hunk 22 above)
Comment (Odinsees) [reply src/styles/scss/abstracts/_mixins.scss]: add this customisation 

example:

@mixin extra-small-text-regular($color: $custom-black) {
  @include custom-text;
  font-weight: $type-weight;
  font-size: 14px;
  line-height: 18px;
  @if $color {
    color: $color;
  }
}

File: src/styles/scss/abstracts/_mixins.scss (hunk 23 above)
Comment (Odinsees) [reply src/styles/scss/abstracts/_mixins.scss]: add default mixin

@mixin custom-text {
  font-family: $type-family;
  font-weight: $type-weight--semi-bold;
  font-style: normal;
}

@mixin extra-small-text-regular($color: $custom-black) {
  @include custom-text;
  font-weight: $type-weight;
  font-size: 14px;
  line-height: 18px;
  @if $color {
    color: $color;
  }
}

File: src/styles/scss/base/_base.scss (hunk 24 above)
Comment (Odinsees) [reply src/styles/scss/base/_base.scss]: remove this

File: src/styles/scss/layout/layout.scss (hunk 25 above)
Comment (Odinsees) [reply src/styles/scss/layout/layout.scss]: in progress in header, footer, and landing story component.

File: tsconfig.json (hunk 26 above)
Comment (Odinsees) [reply tsconfig.json]: fix

File: src/components/common/section-title/styles.module.scss [hunk 27]
```diff
@@ -0,0 +1,16 @@
...
+  padding: 4px 8px;
+
+  background-color: $custom-pink;
+  border-radius: 4px;
+
+  @include custom-text-section;
+  &:first-letter {
+    text-transform: uppercase;
+  }
+}
```
Comment (Ilya-Valasiuk) [inline src/components/common/section-title/styles.module.scss]:
```
.sectionTitle {
  width: fit-content;
  padding: 4px 8px;
  background-color: $custom-pink;
  border-radius: 4px;
 @include custom-text-section;

  &:first-letter {
    text-transform: uppercase;
  }
}
```

File: src/components/home-page/components/stories-list/StoriesList.tsx [hunk 28]
```diff
@@ -1,52 +1,21 @@
...
 
 export const StoriesList: FunctionComponent = () => {
-  const formatDate = (date: Date) => {
-    const formatDate = Moment(date).format('MMM DD, YYYY, HH:mm');
-    return formatDate;
-  };
   const { stories, isLoading, isError } = useStories('technology');
-
+  const isDesktop = useMediaQuery('(min-width: 768px)');
   if (isLoading) return <div>Loading...</div>;
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/StoriesList.tsx]:
```
  const isDesktop = useMediaQuery('(min-width: 768px)');

  if (isLoading) return <div>Loading...</div>;
```

File: src/components/home-page/components/stories-list/story/styles.module.scss [hunk 29]
```diff
@@ -0,0 +1,60 @@
...
+    }
+
+    .storyImage {
+      width: 55%;
+    }
+  }
+}
+
+.storyInfo {
+  padding: 20px 15px;
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/stories-list/story/styles.module.scss]:
```
@media (min-width: $screen-s) {
      display: flex;
      flex-direction: column;
      justify-content: space-between;
      width: 45%;
      padding: 40px;
}
```

File: src/hooks/useArticle.ts [hunk 30]
```diff
@@ -1,7 +1,7 @@
-import { Article } from '@models/article';
+import { Article } from 'models/article';
```
Comment (Ilya-Valasiuk) [inline src/hooks/useArticle.ts]: empty line

File: src/hooks/useStories.ts [hunk 31]
```diff
@@ -1,8 +1,8 @@
-import { TOP_STORIES } from '@constants/api';
+import { TOP_STORIES } from 'constants/api';
 
-import { Story } from '@models/story';
+import { Story } from 'models/story';
 
-import { storiesFetcher } from '@service/storiesFetcher';
+import { storiesFetcher } from 'service/storiesFetcher';
 
 import useSWR from 'swr';
```
Comment (Ilya-Valasiuk) [inline src/hooks/useStories.ts]:
``` 
const { data, error } = useSWR(section, storiesFetcher);
```

File: src/service/storiesFetcher.ts [hunk 32]
```diff
@@ -1,4 +1,4 @@
-import { API_KEY } from '@constants/api';
+import { API_KEY } from 'constants/api';
 
 export const storiesFetcher = async (baseUrl: string) => {
```
Comment (Ilya-Valasiuk) [inline src/service/storiesFetcher.ts]:
```
export const storiesFetcher = async (section: string) => {
```

File: src/styles/scss/abstracts/_mixins.scss [hunk 33]
```diff
@@ -12,5 +12,81 @@
...
+  font-weight: $type-weight--semi-bold;
+  font-style: normal;
+}
+
+@mixin extra-small-text-regular($color: $custom-black) {
+  @include custom-text;
+  font-weight: $type-weight;
+  font-size: 14px;
+  line-height: 18px;
+  @if $color {
```
Comment (Ilya-Valasiuk) [inline src/styles/scss/abstracts/_mixins.scss]: `color: $color;`

File: src/styles/scss/abstracts/_mixins.scss [hunk 34]
```diff
@@ -12,5 +12,81 @@
   font-style: $font-style;
   font-size: $font-size;
   line-height: $line-height;
-  color: $color;
+  color: $custom-black;
+}
+
+@mixin custom-text {
```
Comment (Ilya-Valasiuk) [inline src/styles/scss/abstracts/_mixins.scss]: * 1 mixin for font size
* on html we have font-family

File: src/components/home-page/components/articles-list/ArticlesList.tsx [hunk 35]
```diff
@@ -0,0 +1,31 @@
+import { FunctionComponent } from 'react';
+import { ArticleListItemType } from 'models/articles-list-item';
+import { useArticlesList } from 'hooks/useArticlesList';
+import { MOBILE_SIZE } from 'constants/screenSize';
+import { useMediaQuery } from 'usehooks-ts';
+import { ArticlesListItem } from './articles-list-item/ArticlesListItem';
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/articles-list/ArticlesList.tsx]: double check order

File: src/components/story-page/StoryPage.tsx [hunk 36]
```diff
@@ -1,23 +1,43 @@
 import { FunctionComponent } from 'react';
 import { useRouter } from 'next/router';
-
+import Image from 'next/image';
+import { MOBILE_SIZE } from 'constants/screenSize';
+import { SectionTitle } from 'components/common/section-title/SectionTitle';
 import { useArticle } from 'hooks/useArticle';
+import { formatDate } from 'formatters/date';
+import { useMediaQuery } from 'usehooks-ts';
```
Comment (Ilya-Valasiuk) [inline src/components/story-page/StoryPage.tsx]: double check order

**Review by Ilya-Valasiuk — CHANGES_REQUESTED** [review]

File: src/components/common/section-title/styles.module.scss (hunk 27 above)
Comment (Odinsees) [reply src/components/common/section-title/styles.module.scss]: refactor all scss files

File: src/components/home-page/components/stories-list/StoriesList.tsx (hunk 28 above)
Comment (Odinsees) [reply src/components/home-page/components/stories-list/StoriesList.tsx]: add constant for this

File: src/components/article-page/ArticlePage.tsx [hunk 37]
```diff
@@ -0,0 +1,44 @@
+import { FunctionComponent } from 'react';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { MEDIA_QUERY } from 'constants/screenSize';
+import { BASE_URL_IMAGE } from 'constants/api';
+import { useMediaQuery } from 'usehooks-ts';
```
Comment (Ilya-Valasiuk) [inline src/components/article-page/ArticlePage.tsx]: use useMediaQuerySSR

File: src/components/common/header/components/links/Links.tsx [hunk 38]
```diff
@@ -0,0 +1,57 @@
...
+  setActiveLink,
+  setOpenBurger,
+}) => {
+  const links: LinksType = [
+    { title: 'home', link: '/' },
+    { title: 'world', link: '/world' },
+    { title: 'automobiles', link: '/automobiles' },
+    { title: 'real estate', link: '/real-estate' },
+    { title: 'finance', link: '/world' },
+  ];
```
Comment (Ilya-Valasiuk) [inline src/components/common/header/components/links/Links.tsx]: Please move it out of render function. We don't need to keep this constant in render

File: src/components/common/header/styles.module.scss [hunk 39]
```diff
@@ -0,0 +1,56 @@
...
+//   @include header-shadow;
+
+//   @media (min-width: $screen-s) {
+//     position: inherit;
+//     width: auto;
+//     box-shadow: none;
+//     background-color: inherit;
+//     margin: 0;
+//   }
+// }
```
Comment (Ilya-Valasiuk) [inline src/components/common/header/styles.module.scss]: uncomment or remove it

File: src/components/home-page/components/articles-list/ArticlesList.tsx [hunk 40]
```diff
@@ -0,0 +1,31 @@
+import { FunctionComponent } from 'react';
+import { MEDIA_QUERY } from 'constants/screenSize';
+import { ArticleListItemType } from 'models/articles-list-item';
+import { useArticlesList } from 'hooks/useArticlesList';
+import { useMediaQuery } from 'usehooks-ts';
```
Comment (Ilya-Valasiuk) [inline src/components/home-page/components/articles-list/ArticlesList.tsx]: useMediaQuerySSR

**Review by Ilya-Valasiuk — CHANGES_REQUESTED** [review]

File: src/components/home-page/components/stories-list/story/styles.module.scss (hunk 29 above)
Comment (Odinsees) [reply src/components/home-page/components/stories-list/story/styles.module.scss]: done

File: src/hooks/useArticle.ts (hunk 30 above)
Comment (Odinsees) [reply src/hooks/useArticle.ts]: done

File: src/hooks/useStories.ts (hunk 31 above)
Comment (Odinsees) [reply src/hooks/useStories.ts]: done

File: src/service/storiesFetcher.ts (hunk 32 above)
Comment (Odinsees) [reply src/service/storiesFetcher.ts]: done

File: src/styles/scss/abstracts/_mixins.scss (hunk 33 above)
Comment (Odinsees) [reply src/styles/scss/abstracts/_mixins.scss]: done

File: src/styles/scss/abstracts/_mixins.scss (hunk 34 above)
Comment (Odinsees) [reply src/styles/scss/abstracts/_mixins.scss]: done

File: src/components/article-page/ArticlePage.tsx (hunk 37 above)
Comment (Odinsees) [reply src/components/article-page/ArticlePage.tsx]: done

File: src/components/common/header/components/links/Links.tsx (hunk 38 above)
Comment (Odinsees) [reply src/components/common/header/components/links/Links.tsx]: done

File: src/components/common/header/styles.module.scss (hunk 39 above)
Comment (Odinsees) [reply src/components/common/header/styles.module.scss]: remove

File: src/components/home-page/components/articles-list/ArticlesList.tsx (hunk 40 above)
Comment (Odinsees) [reply src/components/home-page/components/articles-list/ArticlesList.tsx]: done

**Review by Ilya-Valasiuk — APPROVED** [review]
https://media.giphy.com/media/l3q2XhfQ8oCkm1Ts4/giphy.gif

**Comment by Ilya-Valasiuk** [comment]
now please merge it

**Pull request merged**